projects/packages/jetpack-mu-wpcom/src/features/launchpad/class-launchpad-task-lists.php

```diff
@@ -184,9 +189,12 @@ public function is_task_list_dismissible( $id ) {
 	 *
 	 * @param string $id Task List id.
 	 * @param bool   $is_dismissed True if dismissed, false if not.
+	 * @param string $dismissed_until Optional. DateTime object with the date until the task list is dismissed.
 	 */
-	public function set_task_list_dismissed( $id, $is_dismissed ) {
-		$task_list = $this->get_task_list( $id );
+	public function set_task_list_dismissed( $id, $is_dismissed, $dismissed_until ) {
```

Comment:
This feels that we have two behaviors in the same method. Perhaps we should have a dedicated method to set the `dismissed_until` prop.

Reply:
Implemented as part this change https://github.com/Automattic/jetpack/pull/34889/commits/06546f260d0ab9b33806391ca7cd121f79209b6b